api/src/main/java/com/messagebird/objects/integrations/WhatsAppTemplate.java

```diff
@@ -0,0 +1,86 @@
+package com.messagebird.objects.integrations;
+
+import com.messagebird.exceptions.GeneralException;
+import java.util.List;
+
+/**
+ * WhatsApp Template Object as integrations API request.
+ *
+ * @see <a href="https://developers.messagebird.com/api/integrations">Integrations API</a>
+ * @author ssk910
+ */
+public class WhatsAppTemplate {
+
+  private String name;
+  private String language;
```

Comment:
just a note: language is defined as HSMLanguage on the doc but we can keep it as a string in here. No need to update it.